Dockerfile

```diff
@@ -34,11 +34,12 @@ RUN --mount=type=cache,target=/usr/local/cargo/registry \
 RUN --mount=type=cache,target=/usr/local/cargo/registry \
     --mount=type=cache,target=/app/target \
     cargo install \
+    --features tokio-uring \
     --locked \
     --no-default-features \
+    --path ./web3_proxy \
     --profile faster_release \
-    --root /opt/bin \
-    --path ./web3_proxy
+    --root /opt/bin
```

Comment:
latest master has this going to /usr/local/bin. I think this might mean we need 1 more rebase